CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the IBM® Db2® Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fix error messages when host, port, user, password, or database are omitted
+- BugFix: Fix daemon mode prompting
```

Comment:
Can we use past tense in the changelog to be consistent? Sorry for being picky 😢